haskell/gtk-gui/package.yaml

```diff
@@ -0,0 +1,32 @@
+name:                gtk-gui
+version:             0.1.0.0
+github:              "stackbuilders/blog-code"
+license:             BSD-3-Clause
+author:              "Mark Karpov"
+maintainer:          "jgalarza@stackbuilders.com"
+copyright:           "2016 Mark Karpov"
+
+extra-source-files:
+- README.md
+
+description:         Please see the README on GitHub at <https://github.com/githubuser/gtk-gui#readme>
```

Comment:
This description doesn't seem to have a correct link.

Reply:
I have removed it as I do not see it as completely necessary.